clients/osv/src/main/kotlin/Model.kt

```diff
@@ -136,6 +136,7 @@ data class Package(
 data class Range(
     val type: Type,
     val repo: String? = null,
+    @Serializable(EventListSerializer::class)
```

Comment:
@fviernau, FYI, you can verify that deserialization of the old format is still required by removing this line and running `OsvServiceFunTest`. You'll see then that tests already fail deserializing the *actual* results (not the expected results).

Reply:
This was a red herring. I was indeed able to revert most of the changes, see https://github.com/oss-review-toolkit/ort/pull/7444. Sorry @fviernau!